### DISCUSSION TITLE
I love the auto tagging

### DISCUSSION THREAD

**Discussion by Tintalala** [open-webui/open-webui#7449] [Show and tell]
Hi Devs,
I have been asked what chats are being chatted about and the auto tags make it possible.  My Openweb UI setup is with a postgres database and a Grafana container.  Great work thank you!

Using the Word Cloud plugin (magnesium world cloud panel 1.2.11) for Grafana v11.4.0, it looks like this:

![image (5)](https://github.com/user-attachments/assets/8092da8b-f375-49b7-a115-e557a2a5663e)

**Comment by tjbck** [comment]
Looks super cool!

**Comment by flefevre** [comment]
Could you share the Grafana configuration so we could also get this lovely charts?
And does anyone have been able to retrieve tags usage in Langfuse?
